README.md

```diff
@@ -304,12 +302,12 @@ similar to those available for Certificate Transparency logs
 
 In Log mode, Trillian includes an additional Signer component; this component
 periodically processes pending queued items and adds them to the Merkle tree,
-creating a new signed tree head as a result.
+creating a newly signed tree head as a result.
```

Comment:
"signed tree head" is a noun, it's an object returned by Trillian.
"a newly signed" kinda implies the tree head existed before, but it's now been signed.
In this case "a new signed tree head" is indeed created, so I think this was right before.